content/sensu-go/5.0/reference/tokens.md

```diff
@@ -16,32 +16,32 @@ menu:
 When a check is scheduled to be executed by an agent, it first goes through a token substitution step. Any tokens matching attribute values in the check are applied, and then the check is executed. Invalid templates or unmatched tokens will return an error, which is logged and sent to the Sensu backend message transport. Checks with token matching errors will not be executed.
 
 ## New and improved tokens
-Sensu Go uses the [Go template][1] package to implement token substitution. Instead of using triple colons `:::` as in [1.x token substitution][2], Sensu Go token substitution uses double curly braces around the token, and a dot before the attribute to be substituted, such as: `{{ .System.Hostname }}`.
+Sensu Go uses the [Go template][1] package to implement token substitution. Instead of using triple colons `:::` as in [1.x token substitution][2], Sensu Go token substitution uses double curly braces around the token, and a dot before the attribute to be substituted, such as: `{{ .system.hostname }}`.
 
 ## Sensu tokens specification
 
 ### Token substitution syntax
 
-Tokens are invoked by wrapping references to entity or custom attributes with double curly braces, such as `{{ .ID }}` to substitute an entity's ID value. Nested Sensu [entity attributes][3] can be accessed via dot notation (ex: `System.Arch`).
+Tokens are invoked by wrapping references to entity attributes and labels with double curly braces, such as `{{ .name }}` to substitute an entity's name. Nested Sensu [entity attributes][3] can be accessed via dot notation (ex: `system.arch`).
 
-- `{{ .ID }}` would be replaced with the [entity `ID` attribute][3]
-- `{{ .URL }}` would be replaced with a custom attribute called `url`
-- `{{ .Disk.Warning }}` would be replaced with a custom attribute called
+- `{{ .name }}` would be replaced with the [entity `name` attribute][3]
+- `{{ .url }}` would be replaced with a custom label called `url`
+- `{{ .disk.warning }}` would be replaced with a custom label called
```

Comment:
It was my understanding that labels are required to be a flat map of key value pairs and that annotations cannot be accessed by tokens. How should we correct this example?

Reply:
`{{ .labels.disk-warning }}` IMO

Reply:
Perfect. Thanks!